src/Option_Alpha/web/routes/scanner.py

```
@@ -0,0 +1,112 @@
+        {
+            "request": request,
+            "scores": scores,
+            "sort": sort,
+            "direction": direction,
+        },
+    )
+
+
+@router.post("/scanner/run")
+async def run_scan(
+    request: Request,
```

Ruff (0.15.1)

[warning] Unused function argument: `request`

(ARG001)